packages/pds/src/feed-gen/bsky-team.ts

```diff
@@ -0,0 +1,53 @@
+import { NotEmptyArray } from '@atproto/common'
+import { QueryParams as SkeletonParams } from '../lexicon/types/app/bsky/feed/getFeedSkeleton'
+import AppContext from '../context'
+import { paginate } from '../db/pagination'
+import { AlgoHandler, AlgoResponse } from './types'
+import { FeedKeyset } from '../app-view/api/app/bsky/util/feed'
+
+const BSKY_TEAM: NotEmptyArray<string> = [
+  'did:plc:oky5czdrnfjpqslsw2a5iclo', // jay
+  'did:plc:yk4dd2qkboz2yv6tpubpc6co', // daniel
+  'did:plc:ragtjsm2j2vknwkz3zp4oxrd', // paul
+  'did:plc:l3rouwludahu3ui3bt66mfvj', // devin
+  'did:plc:tpg43qhh4lw4ksiffs4nbda3', // jake
+  'did:plc:44ybard66vv44zksje25o7dz', // bryan
+  'did:plc:qjeavhlw222ppsre4rscd3n2', // rose
+  'did:plc:vjug55kidv6sye7ykr5faxxn', // emily
+  'did:plc:fgsn4gf2dlgnybo4nbej5b2s', // ansh
+  'did:plc:vpkhqolt662uhesyj6nxm7ys', // why
+  'did:plc:z72i7hdynmk6r22z27h6tvur', // @bsky.app
+  'did:plc:ewvi7nxzyoun6zhxrhs64oiz', // @atproto.com
+]
+
+const handler: AlgoHandler = async (
+  ctx: AppContext,
+  params: SkeletonParams,
+  requester: string,
+): Promise<AlgoResponse> => {
+  const { limit = 50, cursor } = params
+  const accountService = ctx.services.account(ctx.db)
+  const feedService = ctx.services.appView.feed(ctx.db)
+  const graphService = ctx.services.appView.graph(ctx.db)
+
+  const { ref } = ctx.db.db.dynamic
+
+  const postsQb = feedService
+    .selectPostQb()
+    .where('post.creator', 'in', BSKY_TEAM)
+    .whereNotExists(accountService.mutedQb(requester, [ref('post.creator')]))
+    .whereNotExists(graphService.blockQb(requester, [ref('post.creator')]))
```

Comment:
May just be missing a fine detail on this— but is the mute and block filtering necessary here?  I ask because I expected the user's PDS to do the work to filter-out bad results in `getFeed`.

Reply:
not necessary, but you could get smaller pages without this

basically blocked/muted posts will count against the pagination

feed gens should probably be encouraged to take blocks into account (tho obvs not required)